config/base.yaml

```diff
@@ -1,6 +1,30 @@
 ### Security Server Toolkit Configuration Settings
-base:
-  url: https://ss1:4000/api/v1
-  api_key: X-Road-apikey token=6189108b-7a1d-4bce-a539-265eef3f6a6a
 
+security-server:
+  - name: ss1
```

Comment:
I think it would be better to use placeholders in this file and provide more information about each configuration item in the documentation. For example:

```
  - name: <SECURITY_SERVER_NAME>
    url: https://<SECURITY_SERVER_FQDN_OR_IP>:4000/api/v1
    api_key: X-Road-apikey token=<API_KEY>
    configuration_anchor: /path/to/configuration-anchor.xml
    owner_member_class: <MEMBER_CLASS>
    owner_member_code: <MEMBER_CODE>
    security_server_code: <SERVER_CODE>
    software_token_pin: <SOFT_TOKEN_PIN>
```

Then the documentation would give more information about different items, e.g., `SECURITY_SERVER_NAME`, `SECURITY_SERVER_FQDN_OR_IP`, etc.

In any case, secrets of any existing environments must not be stored on GitHub.

Reply:
Could add YAML comments with placeholders example too, where applicable, to give format example, e.g. 

```yml
    api_key: X-Road-apikey token=<API_KEY> # e.g.: token=eeeeeeee-aaaa-bbbb-cccc-dddddddddddd
```